README.md

```diff
@@ -1,2 +1,7 @@
 # [Sample Repo] PR Practice
 A sample repo for practicing how to create Pull Requests
+
+# Introduction
```

Comment:
Should this be a level 1 heading? 
Please use a level 2 heading here to adhere to hierarchy